cli/pkg/commands/images/describe.go

```diff
@@ -0,0 +1,51 @@
+/*
+Copyright 2024. Open Data Hub Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package images
+
+import (
+	"fmt"
+
+	"github.com/opendatahub-io/ai-edge/cli/pkg/commands/common"
+	. "github.com/opendatahub-io/ai-edge/cli/pkg/commands/common"
+	"github.com/opendatahub-io/ai-edge/cli/pkg/commands/flags"
+	. "github.com/opendatahub-io/ai-edge/cli/pkg/commands/flags"
+	"github.com/spf13/cobra"
+)
+
+var describeCmd = common.NewCmd(
+	fmt.Sprintf(
+		"describe -%s model-id -%s version [-%s model-registry-url] [-%s params-file]",
+		flags.FlagModelID.Shorthand(),
+		flags.FlagVersionName.Shorthand(),
+		flags.FlagModelRegistryURL.Shorthand(),
+		flags.FlagParams.Shorthand(),
+	),
+	"View details of a edge model image.",
+	`View details of a edge model image.
```

Comment:
```suggestion
	"View details of an edge model image.",
	`View details of an edge model image.
```

You added "edge" in here where it wasn't before, so this now has to go back to "an".

Reply:
lol yea my bad, i added it just before pushing and missed out vowels there... changed it now